### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by Isha2806**

### CODE REVIEW

File: Components/Alert/alert.css [hunk 1]
```diff
@@ -1,3 +1,23 @@
-*{
-    background-color: #202122;
-}
\ No newline at end of file
+.alert {
+	padding: 8px;
```
Comment (VedanthB) [inline Components/Alert/alert.css:2]: can you rem here

**Review by VedanthB — APPROVED** [review]
looks good to go 🚀

File: Components/Alert/alert.css (hunk 1 above)
Comment (Isha2806) [reply Components/Alert/alert.css:2]: sure...

Comment (Isha2806) [reply Components/Alert/alert.css]: I have changed it to 1.5 rem...

**Review by tanishk15sharma — APPROVED** [review]
good to merge

**Pull request merged**